### ISSUE TITLE
BiCifParaformer python inference error

### ISSUE THREAD

**Issue by Li-jianqun** [alibaba-damo-academy/FunASR#798]
![image](https://github.com/alibaba-damo-academy/FunASR/assets/56469483/afaefdc1-d075-45c5-96f9-b6859b782410)

**Comment by hnluo** [comment]
Please list your envs:

OS: [e.g. linux]
Python/C++ Version：
Package Version：pytorch、torchaudio、modelscope、funasr version（pip list）
Model：
Command：
Details：
Error log：

**Comment by R1ckShi** [comment]
bicif paraformer模型暂时不支持finetune，同时训练paraformer和cif2的前向没有实现，导致cif2的参数没有训练，预测的token数错误
后续会支持这个模型的finetune

**Comment by R1ckShi** [comment]
it looks like 
   1) cif2 is not well-trained and generats wrong token numbers, can you provide the training detail including loss information?
   2) or ASR model is not well-trained and the recognized tokens contains useless ones including <s> which is filtered out, you may check the variable 'token_int' (funasr/bin/asr_infer.py: line 500) to checkout if the ASR model generates the right characters.
